packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -258,5 +258,25 @@ export const comments: { readonly [key in TokenName]: string | null } = {
     elevation2BoxShadow:
         'The box shadow for elevation 2. Used for components such as menus, banners, tooltips, error notifications, and scrolling.',
     elevation3BoxShadow:
-        'The box shadow for elevation 3. Used for components such as dialogs, overlays, and pop-ups.'
+        'The box shadow for elevation 3. Used for components such as dialogs, overlays, and pop-ups.',
+    bannerGapSize: 'Space between stacked banners',
+    bannerButtonHeight: 'Height of action button in a banner',
+    bannerFontSize: 'Font size for the banner',
+    bannerColor: 'Color for the banner',
+    bannerNeutralBackgroundColor:
+        'Banner background color for no severity value',
+    bannerErrorBackgroundColor: 'Banner background color for error severity',
+    bannerWarningBackgroundColor:
+        'Banner background color for warning severity',
+    bannerInfoBackgroundColor: 'Banner background color for info severity',
+    bannerIconColor: 'Color for icon in banner',
+    bannerIconMarginTop: 'Top margin for icon in banner',
+    bannerStartEndWidth: 'The width of the regions on both ends of the banner',
+    bannerTextMarginVertical: 'Top and bottom margin of text in banner',
+    bannerTextGap: 'Gap between title and body text in banner',
+    bannerButtonFillSelectedColor:
+        'Fill selected color for banner action button',
+    bannerLinkActiveFontColor: 'Color for active link in banner',
+    bannerCloseButtonSize: 'Size of the close button in banner',
```

Comment:
nit: I recall a discussion about "close" vs "dismiss" terminology in your HLD but I can't remember where we landed. Should we change these token names to "dismiss" to match with the component APIs? Or were there cases where we want to use each terminology. 

Same question elsewhere in the PR in function names and the CSS class name.

Reply:
We had settled on using "dismiss" for the spec/docs, but I didn't bother to change the terminology in the internal implementation. I can change these to "dismiss".